src/modules/solution-explorer/solution-explorer-solution/solution-explorer-solution.ts

```diff
@@ -418,6 +425,20 @@ export class SolutionExplorerSolution {
     return this.activeDiagram.uri;
   }
 
+  private _sortDiagramsOfSolution(): void {
+    type DiagramSorter = (firstElement: IDiagram, secondElement: IDiagram) => number;
+
+    const sortOptions: Intl.CollatorOptions = {
+      caseFirst: 'lower',
+    };
+
+    const sorter: DiagramSorter = (firstElement: IDiagram, secondElement: IDiagram): number => {
+        return firstElement.name.localeCompare(secondElement.name, undefined, sortOptions);
```

Comment:
Almost missed it, but there are two spaces too many here 😉 

```suggestion
      return firstElement.name.localeCompare(secondElement.name, undefined, sortOptions);
```